### PR TITLE
fetch data from database, created a series list page and a home page.

### PR DESCRIPTION
**Pull request by Katelyn-Ngo**

### CODE REVIEW

**Comment by Katelyn-Ngo** [comment]
Recording 2024-08-01 at 14 29 36

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,35 +1,24 @@
...
-      <h1>{serverData}</h1>
-    </>
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/series" element={<SeriesList />} />
+        <Route path="*" element={<NotFound />} />
+      </Routes>
+    </BrowserRouter>
```
Comment (shawnkost) [inline client/src/App.tsx]: `<BrowserRouter>` should be inside of `index.tsx`. You wrap it around `<App />`

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,35 +1,24 @@
...
-        </a>
-        <a href="https://react.dev" target="_blank" rel="noreferrer">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>{serverData}</h1>
-    </>
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
```
Comment (shawnkost) [inline client/src/App.tsx]: Works, but if you want the `<Navbar />` to be visible on every page, that's where the `<Outlet />` comes into play. So your `<Navbar>` should be the parent `route` to all of these

File: client/src/pages/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+// import React from 'react';
+// import { Link } from 'react-router-dom';
```
Comment (shawnkost) [inline client/src/pages/HomePage.tsx]: Delete unused code

File: client/src/pages/SeriesList.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  const [loading, setLoading] = useState(true);
+  const [err, setErr] = useState<unknown>();
+
+  useEffect(() => {
+    async function getSeries() {
+      try {
+        const response = await fetch('/api/series');
+        if (!response.ok) throw new Error(`Response status ${response.status}`);
+        const series = (await response.json()) as Series[];
+        console.log(series);
```
Comment (shawnkost) [inline client/src/pages/SeriesList.tsx]: Remove all `console.log`s

File: package.json [hunk 5]
```diff
@@ -44,5 +44,11 @@
     "stylelint-config-standard": "^32.0.0",
     "tsx": "^4.7.0",
     "typescript": "^5.4.3"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.6.0",
+    "@fortawesome/free-solid-svg-icons": "^6.6.0",
+    "@fortawesome/react-fontawesome": "^0.2.2",
+    "react-router-dom": "^6.25.1"
```
Comment (shawnkost) [inline package.json]: Any react packages should be installed in the `client` package.json, not the one at the root

**Review by shawnkost — COMMENTED** [review]
Looking great so far, left some comments

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**